frontend/src/components/nodes/ValidatorCollapsedRow.tsx

```diff
@@ -3,27 +3,36 @@ import * as React from "react";
 import { TableCollapseRow } from "../utils/Table";
 
 import ValidatorMetadataRow from "./ValidatorMetadataRow";
-import ValidatorTelemetryRow from "./ValidatorTelemetryRow";
+import ValidatorProgressElement from "./ValidatorProgressElement";
+import ValidatorTelemetryElements from "./ValidatorTelemetryElements";
 import {
-  NodeInfo,
-  PoolDetails,
+  ValidatorTelemetry,
+  ValidatorDescription,
   ValidationProgress,
 } from "../../libraries/wamp/types";
+import { ValidatorNodesContentRow } from "./ValidatorRow";
 
 interface Props {
   isRowActive: boolean;
   progress?: ValidationProgress;
-  nodeInfo?: NodeInfo;
-  poolDetails?: PoolDetails;
+  telemetry?: ValidatorTelemetry;
+  description?: ValidatorDescription;
 }
 
 const ValidatorCollapsedRow: React.FC<Props> = React.memo(
-  ({ isRowActive, progress, nodeInfo, poolDetails }) => (
+  ({ isRowActive, progress, telemetry, description }) => (
     <TableCollapseRow collapse={isRowActive}>
       <td colSpan={8}>
-        <ValidatorTelemetryRow progress={progress} nodeInfo={nodeInfo} />
+        <ValidatorNodesContentRow noGutters>
+          <link
+            href="https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@500&display=swap"
+            rel="stylesheet"
+          />
```

Comment:
Do we only need it here? Is it some sort of an artifact?

Reply:
Actually, we need it one level deeper - in `ValidatorTelemetryElements.tsx`, as it's the only place that uses that font.